content/language-specific/chinese.markdown

```diff
@@ -0,0 +1,121 @@
+##  [37] "已"       "发展"     ""         "野蛮"     "暴行"     ""         ""         "暴行"     "玷污"    
+##  [46] "了"       "人类"     ""         "良心"     ""         ""         "一个"     "人人"     "享有"    
+##  [55] "言论"     ""         "信仰"     "自由"     ""         "免"       "予"       "恐惧"     ""        
+##  [64] "匮"       "乏"       ""         "世界"     ""         "来临"     ""         "已"       "被"      
+##  [73] "宣布"     ""         "普通"     "人民"     ""         "最高"     "愿望"     ""         "鉴于"    
+##  [82] ""         "使"       "人类"     "不致"     "迫不得已" "铤"       ""         "走"       "险"      
+##  [91] ""         "暴政"     ""         "压迫"     "进行"     "反叛"     ""         ""         "必要"    
+## [100] "使"       "人权"     ""         "法治"     ""         "保护"     ""         "鉴于"     ""        
+## [109] "必要"     "促进"     "各国"     "间"       "友好"     "关系"     ""         "发展"     ""        
+## [118] "鉴于"     "各"       "联合"     "国"       "国家"     ""         "人民"     "已"       ""        
+## [127] "联合"     "国"       "宪章"     ""         "重申"     "他们"     ""         "基本"     "人权"    
+## [136] ""         "人格"     "尊严"     ""         "价值"     ""         "男女平等" "权利"     ""        
+## [145] "信念"     ""         ""         "决心"     "促成"     "较大"     "自由"     "中的"     "社会"    
+## [154] "进步"     ""         "生活"     "水平"     ""         "改善"     ""         "鉴于"     "各"      
+## [163] "会员"     "国"       "业"       "已"       "誓"       "愿"       ""         "联合"     "国"      
+## [172] "合作"     ""         "促进"     ""         "人权"     ""         "基本"     "自由"     ""        
+## [181] "普遍"     "尊重"     ""         "遵行"     ""         "鉴于"     ""         ""         "权利"    
+## [190] ""         "自由"     ""         "普遍"     "了解"     ""         ""         "誓"       "愿"      
+## [199] ""         "充分"     "实现"     "具有"     "很大"     ""         "重要性"   ""         ""        
+## [208] "现在"     ""         "大会"     ""         "发布"     ""         ""         "世界"     "人权"    
+## [217] "宣言"     ""         ""         ""         "人民"     ""         ""         "国家"     "努力"    
+## [226] "实现"     ""         "共同"     "标准"     ""         "以期"     "每一个人" ""         "社会"    
+## [235] "机构"     "经常"     "铭"       "念"       "本"       "宣言"     ""         "努力"     ""        
+## [244] "教诲"     ""         "教育"     "促进"     ""         "权利"     ""         "自由"     ""        
+## [253] "尊重"     ""         ""         ""         "国家"     ""         ""         "国际"     ""        
+## [262] "渐进"     "措施"     ""         "使"       ""         "权利"     ""         "自由"     ""        
+## [271] "各"       "会员"     "国"       "本身"     "人民"     ""         ""         ""         "管辖"    
+## [280] ""         "领土"     ""         "人民"     ""         "得到"     "普遍"     ""         "有效"    
+## [289] ""         "承认"     ""         "遵行"     ""
+```
+
+Also, you can create your own stopword list.
+
+
+```r
+stwords <- c("", "已", "了", "使")
```

Comment:
@yuanzhouIR if "已", "了", "使" should be remove always, it is better to update marimo itself (better not encourage people to cherry pick words). Can you do?

I apply `tokens_select(toks, min_nchar = 2)` in Japanese. Do you think it is too aggressive?

Reply:
Probably, this is a good approach.
```
tokens_select('^\\p{Han}+$', valuetype = 'regex', min_nchar = 2, padding = TRUE)
```

Reply:
> Probably, this is a good approach.
> 
> ```
> tokens_select('^\\p{Han}+$', valuetype = 'regex', min_nchar = 2, padding = TRUE)
> ```

I think it is OK to do so because in modern Chinese language, most words are disyllable and a single Chinese character is not meaningful in most cases.

Reply:
Great to know!